features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/tracking/AndroidXFragmentLifecycleCallbacks.kt

```diff
@@ -94,4 +112,8 @@ internal open class AndroidXFragmentLifecycleCallbacks(
     }
 
     // endregion
+
+    companion object {
+        private const val STOP_VIEW_DELAY_MS = 200L
```

Comment:
I see that this constant is defined here but also in other places. Should we unify to a SSOT incase we want to modify the delay later on?